src/prisma/schema.prisma

```diff
@@ -11,7 +11,25 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  email    String @unique
+  id       Int      @id @default(autoincrement())
+  email    String   @unique
   password String
+  groups   Member[]
+}
+
+model Member {
+  user_id  Int
+  user     User  @relation(fields: [user_id], references: [id], onDelete: Cascade)
+  group_id Int
+  group    Group @relation(fields: [group_id], references: [id], onDelete: Cascade)
```

Comment:
Do we also want to delete the group when deleting a member? Confirm if this is the intended behaviour please